CONTRIBUTING.md

```diff
@@ -141,6 +141,7 @@ To start contributing to the project you would need to set up the project in you
   - `./scripts/check-all-the-things.sh` on Linux and MacOS
   - `.\scripts\check-all-the-things.ps1` on Windows
 
+**Important:** Node v12.19 is required.
```

Comment:
I would say that this is the minimim required version, because it sounds that we are supporting just this version

Reply:
Single-click suggestion:
```suggestion
**Important:** Node version v12.19 or greater is required.
```

Reply:
It looks like that's actually the case :trollface: https://github.com/boostercloud/booster/issues/494

Reply:
I have node v14 and everything runs well.

In any case, I would add the "minimum" part and then try to fix the problems with node 15

Reply:
Yeah, it's weird that something works only under a specific version, ideally it should work starting in some specific version and in all the next...